### PR TITLE
Remove a dead assignment in CondFormats/SiStripObjects/src/SiStripThreshold.cc

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

Trivial PR only intended to silence a warning issued by the static analyzer.
I also profit to reformat in a more readable way a comment line in it.

#### PR validation:

It compiles.
Nothing relevant is modified, nothing should change in outputs.

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: CondFormats/SiStripObjects/src/SiStripThreshold.cc [hunk 1]
```diff
@@ -143,19 +143,14 @@ void SiStripThreshold::printSummary(std::stringstream& ss, const TrackerTopology
...
         stripRange = (itp->getFirstStrip() - firstStrip);
       else
-        stripRange =
-            firstStrip > 511
-                ? 768 - firstStrip
-                : 512 -
-                      firstStrip;  //*FIXME, I dont' know ithis class the strip number of a detector, so I assume wrongly that if the last firstStrip<511 the detector has only 512 strips. Clearly wrong. to be fixed
-
+        //*FIXME, I dont' know ithis class the strip number of a detector, so I assume wrongly that if the last firstStrip<511 the detector has only 512 strips. Clearly wrong. to be fixed
+        stripRange = firstStrip > 511 ? 768 - firstStrip : 512 - firstStrip;
```
Comment (mmusich) [inline CondFormats/SiStripObjects/src/SiStripThreshold.cc:153]: Just for the record, I *think* one could make this exact by means of using the `StripTopology` (you can get it from this method

https://github.com/cms-sw/cmssw/blob/6d26b89d99a92507aad02623bd3feac40f65d369/Geometry/TrackerGeometryBuilder/interface/StripGeomDetUnit.h#L41
)
the con is that one would need to pass a `TrackerGeometry` in input to `SiStripThreshold::printSummary`.
An alternative way is to use the mechanism from

https://github.com/cms-sw/cmssw/blob/6d26b89d99a92507aad02623bd3feac40f65d369/CalibFormats/SiStripObjects/interface/SiStripDetInfo.h#L55

and use the fact that

```C++
unsigned int nStrip = detInfo_.getNumberOfApvsAndStripLength(detid).first * sistrip::STRIPS_PER_APV;
```

File: CondFormats/SiStripObjects/src/SiStripThreshold.cc [hunk 2]
```diff
@@ -143,19 +143,14 @@ void SiStripThreshold::printSummary(std::stringstream& ss, const TrackerTopology
...
       if (itp != eit)
         stripRange = (itp->getFirstStrip() - firstStrip);
       else
-        stripRange =
-            firstStrip > 511
-                ? 768 - firstStrip
-                : 512 -
-                      firstStrip;  //*FIXME, I dont' know ithis class the strip number of a detector, so I assume wrongly that if the last firstStrip<511 the detector has only 512 strips. Clearly wrong. to be fixed
-
+        //*FIXME, I dont' know ithis class the strip number of a detector, so I assume wrongly that if the last firstStrip<511 the detector has only 512 strips. Clearly wrong. to be fixed
```
[the remark this answers is not in the document]
Comment (perrotta) [reply CondFormats/SiStripObjects/src/SiStripThreshold.cc]: Rewritten in (hopefully) some better English

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**